db/migrate/20210115231123_add_email_reviewers_status_changed_to_collection.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailReviewersStatusChangedToCollection < ActiveRecord::Migration[6.0]
+  def change
+    add_column :collections, :email_reviewers_status_changed, :boolean, null: true
```

Comment:
I'm not sure I understand why this was added.

Reply:
Following this: https://github.com/sul-dlss/happy-heron/blob/main/db/structure.sql#L159 ..

Reply:
<img width="881" alt="Screen Shot 2021-01-19 at 9 53 04 AM" src="https://user-images.githubusercontent.com/92044/105111589-ce686e00-5a86-11eb-81d3-9a230be16492.png">
That is there to drive this UI element.

Reply:
Won't I need to do something similar: https://github.com/sul-dlss/happy-heron/blob/main/app/services/collection_observer.rb#L29  ?

Reply:
I don't think so.